### ISSUE TITLE
make error in publications directory 

### ISSUE THREAD

**Issue by weijianzhang** [JuliaLang/julialang.github.com#319]
Hi, 

When I ran `make` in the `pulications` directory, I got the following error:
```
$ make
PATH=/home/weijian/anaconda3/bin:/home/weijian/anaconda/bin:/home/weijian/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/bin pandoc --reference-links -s -S --biblio julia.bib --csl jlwebpage.csl -o index_new.html _EDIT_ME_index.md
sed: can't read : No such file or directory
make: *** [all] Error 2
```
It seems `sed` can not find `index_new.html`. Note I have both pandoc and pandoc-citeproc installed.

Could you let me know how I can fix this error? 
 
Thanks a lot,

Weijian

**Comment by jiahao** [comment]
It looks like the sed syntax I used only works on BSD/Darwin sed, not on Linux. I'll commit the fix after I check that it still works on OSX.

**Comment by weijianzhang** [comment]
Thanks a lot.

**Comment by jiahao** [comment]
Fixed by #338
